vision_msgs_rviz_plugins/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>vision_msgs_rviz_plugins</name>
+  <version>0.1.0</version>
+  <description>RVIZ2 plugins for visualizing vision_msgs</description>
+  <maintainer email="georg.novotny@aon.at">Georg Novotny</maintainer>
```

Comment:
Go ahead and put me too so I can get build farm failure messages if there are problems